src/devices/bus/centronics/epson_lx810l.h

```diff
@@ -107,30 +108,21 @@ class epson_lx810l_device : public device_t, public device_centronics_peripheral
 
 	void lx810l_mem(address_map &map);
 
-	/* Video hardware (simulates paper) */
-	uint32_t screen_update_lx810l(screen_device &screen, bitmap_rgb32 &bitmap, const rectangle &cliprect);
-
-	unsigned int bitmap_line(int i) { return ((std::abs(m_pf_pos_abs) / 6) + i) % m_bitmap.height(); }
-
 	required_device<cpu_device> m_maincpu;
-	required_device<stepper_device> m_pf_stepper;
-	required_device<stepper_device> m_cr_stepper;
+	required_device<bitmap_printer_device> m_bitmap_printer;
 	required_device<eeprom_serial_93cxx_device> m_eeprom;
 	required_device<e05a30_device> m_e05a30;
-	required_device<screen_device> m_screen;
 
 	output_finder<> m_online_led;
 
 	int m_93c06_clk;
 	int m_93c06_cs;
 	uint16_t m_printhead;
-	int m_pf_pos_abs;
-	int m_cr_pos_abs;
-	int m_real_cr_pos;
 	int m_real_cr_steps;
-	int m_real_cr_dir; /* 1 is going right, -1 is going left */
 	uint8_t m_fakemem;
-	bitmap_rgb32 m_bitmap;
+	int m_in_between_offset = 0; // in between cr_stepper phases
```

Comment:
If these default values are important, they should be set in the device_start/device_reset members (depending on if reset actually changes them).